README.md

```diff
@@ -1,113 +1,58 @@
 
-#### Setup Assemblyline config file
+This repo is dedicated for the new version of Assemblyline 4 UI. It uses the React library and Material UI for the visual components.
 
-From the `docker` directory, copy the file `config.yml.template` to `config.yml` in the same directory.
+## Image variants and tags
 
-Change the `<YOUR_IP>` in the newly created `config.yml`file to the IP of your dev machine.
+| **Tag Type** | **Description**                                                                                  |      **Example Tag**       |
+| :----------: | :----------------------------------------------------------------------------------------------- | :------------------------: |
+|    latest    | The most recent build (can be unstable).                                                         |          `latest`          |
+|  build_type  | The type of build used. `dev` is the latest unstable build. `stable` is the latest stable build. |     `stable` or `dev`      |
+|    series    | Complete build details, including version and build type: `version.buildType`.                   | `4.5.stable`, `4.5.1.dev3` |
 
-#### Setup Assemblyline classification engine file
+#### Running this component
 
-From the `docker` directory, copy the file `classification.yml.template` to `classification.yml` in the same directory.
+```bash
+docker run --name frontend cccs/assemblyline-ui-frontend
+```
+## Development
 
-Change the `enforce` value to `true` in the `classification.yml` file to turn on the classification engine.
+For information on frontend development, follow this [guide](https://cybercentrecanada.github.io/assemblyline4_docs/developer_manual/frontend/frontend/).
 
-## Launch the dev environment
+## Documentation
 
-### Dependencies
+For more information about this Assemblyline component, follow this [overview](https://cybercentrecanada.github.io/assemblyline4_docs/overview/architecture/) of the system's architecture.
 
-Go to the `docker` directory and run the following command to launch the Assemblyline DB and UI.
+---
 
-    docker-compose up
+# Front-end de l'interface utilisateur d'Assemblyline
 
-### Frontend
+Ce repo est dédié à la nouvelle version d'Assemblyline 4 UI. Elle utilise la bibliothèque React et Material UI pour les composants visuels.
 
-Simply use the `npm start script` to launch the frontend.
+## Variantes d'images et balises
 
-### Once dependencies and frontend started
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
```

Comment:
Should be either just in french or english not both. I feel like it should be only in french in this case so it would be "type de production" et "séries".